src/network/MissionControlTasks.cpp

```diff
@@ -141,6 +142,23 @@ void CameraStreamTask::closeStream(const CameraID& cam) {
 	_camera_encoders.erase(cam);
 }
 
+void CameraStreamTask::sendCurrentFrame(const robot::types::CameraID& cam) {
```

Comment:
Why do we need this method? Why can't the callback handler just call `robot::readCamera()`? We don't necessarily need to check `_open_streams` to see if the given stream has been opened or not, we can just read the frame from the camera.